themes/origins/assets/component-slider.js

```diff
@@ -0,0 +1,216 @@
+class Slider extends HTMLElement {
+  static observedAttributes = ["autoplay", "interval", "responsive"];
+
+  constructor() {
```

Comment:
Since this is a custom element that will be used in multiple section, might be worth checking if it has been defined before or not